README.md

```diff
@@ -319,8 +319,12 @@ more information about the governance of the Node.js project, see
 * [zkat](https://github.com/zkat) -
 **Kat Marchán** &lt;kzm@sykosomatic.org&gt;
 
-Collaborators & CTC members follow the [COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md) in
-maintaining the Node.js project.
+All of the CTC members are also Collaborators, we use Collaborators usually 
+to include CTC members as well.
+
+Therefore Collaborators(include CTC members) would follow the 
```

Comment:
Something is wrong on this line. Perhaps, `Collaborators (including CTC members)`?